source/principles/data/index.html.md.erb

```diff
@@ -0,0 +1,187 @@
+
+## Data is Analysable
+
+We need to make sure that we are mindful of how the data we gather can be used to drive analysis, and ensure that enough contextual information is gathered to be able to make sense of the data and link it to data from other services.
+
+## Data use is Ethical
+
+It’s uncontroversial to state that data use must be ethical, but it’s not always obvious what ethical issues might arise. Regulations such as GDPR provide some ideas, but **we must go above and beyond that bare minimum**, and always ensure that our use of data does not compromise our mission: education. If we set a poor example of how data should be used, then we have failed.
+
+> For example, our use of data must never be more than what the person the data is about has knowingly consented to (assuming that people don’t actually read privacy statements and other fine print).
+>
+> If we ask for their address to send them a letter, and the user interface says “Please enter the address you’d like us to send your letter to”, then aggregating that address data to work out which regions of the country use our services the most is probably fine (IF done in compliance with all regulations, which means it’ll need to be mentioned in the privacy policy), because people generally assume that sort of thing might happen.
+>
+> But using it to also send them a birthday card every year, even if it mentions that somewhere in the privacy policy, wouldn’t be ethical. Even if we think it’s a lovely idea, some people don’t want to be reminded of their birthday, and the user interface did not make it clear that was going to happen when they entered their address, so the recipient might find it an unwelcome surprise.
+
+## Rules
+
+We are bound by the following regulations, and you should make yourself familiar with them:
+
+- [GDPR](https://www.gov.uk/government/publications/guide-to-the-general-data-protection-regulation) (General Data Protection Regulation)
+- [Data Protection Act](https://www.gov.uk/government/publications/data-protection-act-2018-overview)
+- [Freedom of Information Act](https://www.legislation.gov.uk/ukpga/2000/36/contents)
+- [DfE Data Policies](https://educationgovuk.sharepoint.com/sites/lvewp00053/SitePages/Policies-and-Standards.aspx)
+- Service teams are responsible for performing the DPIA (Data Protection Impact Assessment) for sensitive data they collect, including its further storage and processing as part of larger projects such as service line analytics.
+- The Data Infrastructure team, however, are responsible for security of that data once it has entered cross-team infrastructure. Therefore, they must provide information to the service teams to enable them to correctly assess the data protection impact of handing data over.
+
+## Guidelines
```

Comment:
I think it's helpful to have guidance here, as a follow-on from the principles, but might need some clearer section segregation so folk reading know they're moving from a principle, to how to implement it. I contemplated whether this should be split out into the guidance section... but probably not.

Reply:
ah ha @govtim, similar thoughts